source/MaterialXCore/Traversal.cpp

```diff
@@ -9,19 +9,31 @@
 
 MATERIALX_NAMESPACE_BEGIN
 
-const Edge NULL_EDGE(nullptr, nullptr, nullptr);
+const Edge& getNullEdge() {
```

Comment:
Just adding a reminder to run `clang-format` on your new code, so that it inherits the Allman braces style of the project.